SatisfactorySaveEditor/ViewModel/MainViewModel.cs

```diff
@@ -523,8 +529,32 @@ private async Task LoadFile(string path)
 
         private void LoadFileAsync(string path)
         {
-            //MessageBox.Show("Loading file...");
-            saveGame = new SatisfactorySave(path);
+            try
+            {
+                saveGame = new SatisfactorySave(path);
+            }
+            catch (Exception ex)
+            {
+                if (ex is FileNotFoundException)
```

Comment:
Don't put an if like this inside the catch, have one `catch(FileNotFoundException ex)` and one `catch(Exception ex)` after it